shared/TestStandComm/SerialTransport.h

```diff
@@ -0,0 +1,41 @@
+#ifndef SERIAL_TRANSPORT_H
+#define SERIAL_TRANSPORT_H
+
+#include "SerialDevice.h"
+#include "Messages.h"
+
+typedef enum {
+    MSG_SEG_START    = 0,
+    MSG_SEG_ID       = 1,
+    MSG_SEG_LENGTH   = 2,
+    MSG_SEG_DATA     = 3,
+    MSG_SEG_CRC      = 4,
+    MSG_SEG_END      = 5
+} MessageSegment;
+
+typedef struct {
+    MessageSegment current_segment;
+    uint8_t bytes_read;
+    uint8_t msg_length;
+    uint16_t crc;
+} PendingMessage;
+
```

Comment:
Would it make sense to migrate `MessageSegment` and `PendingMessage` also to `Messages.h`?

Reply:
I'd rather keep theme here, since they're exclusively used by SerialTransport.h and SerialTransport.cxx.
Someone who uses Messages.h doesn't need access to these structs